lib/screens/app/wallet/custom_transaction.dart

```diff
@@ -64,7 +64,7 @@ class _CustomTransactionState extends State<CustomTransaction> {
       statusStream: _statusNotifier.stream,
       messageStream: _messageNotifier.stream,
       onClose: () {
-        Navigator.of(context).pop();
+        Navigator.of(context).maybePop();
```

Comment:
We don't know where in the app this will show, so seemed better. 

But it was more like a thing that happened during debugging when I made it work and may not be needed in the cases I was testing. 

But I don't feel like testing all screens again just to put pop() back there.